Firestore/core/src/core/field_filter.cc

```diff
@@ -81,30 +82,32 @@ const char* CanonicalName(Filter::Operator op) {
 
 }  // namespace
 
-FieldFilter FieldFilter::Create(const FieldPath& path,
-                                Operator op,
-                                google_firestore_v1_Value value_rhs) {
-  model::SortFields(value_rhs);
+FieldFilter FieldFilter::Create(
+    const FieldPath& path,
+    Operator op,
+    SharedMessage<google_firestore_v1_Value> value_rhs) {
+  google_firestore_v1_Value& value = value_rhs.operator*();
```

Comment:
You can just do `*value_rhs`, no need to call `operator*` explicitly.

Reply:
Done